src/legacy/core_plugins/metrics/server/lib/vis_data/annorations/build_request_body.js

```diff
@@ -16,12 +16,11 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import buildProcessorFunction from '../build_processor_function';
+import processors from '../request_processors/annotations';
 
-import buildProcessorFunction from './build_processor_function';
-import processors from './request_processors/annotations';
-
-export default function buildAnnotationRequest(req, panel, annotation,  esQueryConfig, indexPattern) {
-  const processor = buildProcessorFunction(processors, req, panel, annotation,  esQueryConfig, indexPattern);
+export default function buildAnnotationRequest(...args) {
```

Comment:
I do not see a necessity to add jsdoc here, since all arguments just pass through this function call to the next function in the same order. But anyway jsdoc was added